### DISCUSSION TITLE
Advancements

### DISCUSSION THREAD

**Discussion by Mdbgamer** [BlasterM5/ponyorigins#6] [Q&A] [answered]
Some of the advancement descriptions are rather vague in regards to the Unicorn, and I'm not sure if it's just me, but I can't seem to get the Tekekinetic and Thaumic mastery to trigger. What am I doing wrong/misunderstanding? And while we're at it, is there a list of the advancements and how to achieve them anywhere?

**Comment by BlasterM5** [comment, marked as answer]
Still working on testing the unicorn mastery, have been streamlining and rebuilding the powers for the upcoming 1.20.1 upgrade. Currently, telekinetic is simply by mining blocks outside your normal reach (may change this in the future), and thaumatic is simply by consuming a certain amount of energy.

I do plan on filling out the wiki with much more detail, just haven't gotten around to it yet as I focus on the next update.

**Reply by Mdbgamer** [reply]
Fair enough! I'll try to test that out since I know exactly what should be happening now, but I'll let you know if it's still not working.

**Reply by Mdbgamer** [reply]
Yeahhh, it seems the two masteries might be bugged. Mined out a good 20+ block stretch outside of the base reach, and expended my energy several times times in a row. Nothin.

**Comment by BlasterM5** [comment]
It takes more than 20 blocks to gain the mastery. A lot more than 20. Focus on beating the Ender dragon, the masteries should pop by the time you do.

**Reply by Mdbgamer** [reply]
Fair enough!
